### DISCUSSION TITLE
Schedule as calendar

### DISCUSSION THREAD

**Discussion by vlx** [marcokreeft87/formulaone-card#113] [Ideas]
Hi @marcokreeft87 , I really admire what you do here and I cant wait the new F1 season to start and bring life to Formula1 cards in HA :) Great job!
What do you and the community think about having the schedule card represented as a caledar widget too?  Or import events to the built in HA calendar? It's just an idea to show racing events events in a calendar as well.

**Comment by marcokreeft87** [comment]
It can't be done, because its a frontend card and not an integration. What I do is use the ics integration to import the schedule into a calendar
